templates/test/ci/prow-dalec-custom-builds/patches/control-plane-custom-builds.yaml

```diff
@@ -40,4 +56,8 @@
   path: /spec/kubeadmConfigSpec/preKubeadmCommands/-
   value:
     bash -c /tmp/replace-k8s-binaries.sh
+- op: add
+  path: /spec/kubeadmConfigSpec/initConfiguration/nodeRegistration/ignorePreflightErrors
+  value:
+    - HTTPProxyCIDR
   
```

Comment:
Newline?

Reply:
Do you mean I should remove the existing newline? Or add one somewhere

Reply:
I see the newline at my end so I think it's okay?

<img width="1045" height="457" alt="Image" src="https://github.com/user-attachments/assets/75c8cd67-2e95-4fe1-b534-55bd8782bb02" />